tests/php/Configuration/Parser/MenuParserTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bolt\Tests\Configuration\Parser;
+
+use Bolt\Configuration\Parser\MenuParser;
+use PHPUnit\Framework\TestCase;
+use Tightenco\Collect\Support\Collection;
+
+class MenuParserTest extends TestCase
+{
+    /** @var MenuParser */
+    private $menuParser;
+
+    protected function setUp(): void
+    {
+        $this->menuParser = new MenuParser();
+    }
+
+    public function testCanParse(): void
+    {
+        $config = $this->menuParser->parse();
+
+        $this->assertInstanceOf(Collection::class, $config);
+    }
+
+    public function testHasMenu(): void
+    {
+        $config = $this->menuParser->parse();
```

Comment:
it dies not take any input, so how it parses something?
(maybe file path should be set as constructor parameter?)

Reply:
I'm not sure we should do that.. `menu.yaml` is fixed for now.. Maybe one day we'll allow to configure that, but for now: YAGNI

Reply:
It's not about configuration but clean code. Otherwise it's not a parser but generator.

Reply:
Added a @todo for this.  See #379